### PR TITLE
PVAYLADEV-878 Use case documents changed to md format

### PR DESCRIPTION
**Pull request by LasseMatikainen**
Some links seem to be broken still. "[]{#Ref_IG-CS .anchor}\[IG-CS\]"
Not sure where they are supposed to link to as the word document links did not seem to be working for me as well.

### CODE REVIEW

File: doc/UseCases/uc-gconf_x-road_use_case_model_for_global_configuration_distribution_1.4_Y-883-8.md [hunk 1]
```diff
@@ -0,0 +1,1918 @@
...
+8.  System logs the event “Generate internal configuration signing key”
+    or “Generate external configuration signing key”, depending of the
+    configuration source, to the audit log.
+
+9.  System generates the configuration anchor for the configuration
+    source: 2.2.17.
+
+**Extensions**:
+
+3a. The desired token is not on the list:
```
Comment (tmkrepo) [inline doc/UseCases/uc-gconf_x-road_use_case_model_for_global_configuration_distribution_1.4_Y-883-8.md]: Missing ul from at `3a.` and quotation notation at `3a.1.`

File: doc/UseCases/uc-gconf_x-road_use_case_model_for_global_configuration_distribution_1.4_Y-883-8.md [hunk 2]
```diff
@@ -0,0 +1,1918 @@
...
+    - 4a.1. System displays the error message: “Configuration anchor upload failed: unexpected instance identifier found in anchor”.
+    - 4a.2. System logs the event “Upload configuration anchor failed” to the audit log.
+    - 4a.3. SS administrator selects to reinsert the path to the configuration anchor file. Use case continues from step 3.
+        - 4a.3a. SS administrator selects to terminate the use case.
+
+- 6a. SS administrator cancels the import.
+    - 6a.2. Use case terminates.
+
+- 7a. Downloading of the internal configuration fails.
+    - 7a.1. System displays the error message: “Configuration source cannot be reac
```
Comment (tmkrepo) [inline doc/UseCases/uc-gconf_x-road_use_case_model_for_global_configuration_distribution_1.4_Y-883-8.md]: Number missing, this causes the content listing to not work and the last number to incorrectly be 2.3.6

File: doc/UseCases/uc-service_x-road_use_case_model_for_service_management_1.6_Y-883-3.md [hunk 3]
```diff
@@ -0,0 +1,2578 @@
...
+    * [3.1.4 UC SERVICE\_03: Add a Service Client to a Security Server Client](#314-uc-service_03-add-a-service-client-to-a-security-server-client)
+    * [3.1.5 UC SERVICE\_04: Add Access Rights for a Service Client](#315-uc-service_04-add-access-rights-for-a-service-client)
+    * [3.1.6 UC SERVICE\_05: Remove Access Rights from a Service Client](#316-uc-service_05-remove-access-rights-from-a-service-client)
+    * [3.1.7 UC SERVICE\_06: View the WSDLs of a Security Server Client](#317-uc-service_06-view-the-wsdls-of-a-security-server-client)
+    * [3.1.8 UC SERVICE\_07: View the Services of a Security Server Client](#318-uc-service_07-view-the-services-of-a-security-server-client)
+    * [3.1.9 UC SERVICE\_08: Add a WSDL to a Security Server Client](#319-uc-service_08-add-a-wsdl-to-a-security-server-client)
+    * [3.1.10 UC SERVICE\_09: Edit the Address of a WSDL](#3110-uc-service_09-edit-the-address-of-a-wsdl)
+    * [3.1.11 UC SERVICE\_10: Download and Parse WSDL](#3111-uc-service_10-download-and-parse-wsdl)
+    * [3.1.12 UC SERVICE\_44: Validate a WSDL](#3112-uc-service_44-validate-a-wsdl)
```
Comment (tmkrepo) [inline doc/UseCases/uc-service_x-road_use_case_model_for_service_management_1.6_Y-883-3.md]: Numbered links broken from 3.1.12 to 3.1.31

File: doc/UseCases/uc-ss_x-road_use_case_model_for_security_server_management_1.4_Y-883-4.md [hunk 4]
```diff
@@ -0,0 +1,2761 @@
...
+  * [3.12 UC SS\_11: Generate a New TLS Key and Certificate for the Security Server](#312-uc-ss_11-generate-a-new-tls-key-and-certificate-for-the-security-server)
+  * [3.13 UC SS\_12: Export the Internal TLS Certificate of the Security Server](#313-uc-ss_12-export-the-internal-tls-certificate-of-the-security-server)
+  * [3.14 UC SS\_13: View the List of Configuration Backup Files](#314-uc-ss_13-view-the-list-of-configuration-backup-files)
+  * [3.15 UC SS\_14: Back Up Configuration](#315-uc-ss_14-back-up-configuration)
+  * [3.16 UC SS\_15: Restore Configuration from a Backup File](#316-uc-ss_15-restore-configuration-from-a-backup-file)
+  * [3.17 UC SS\_16: Download a Backup File](#317-uc-ss_16-download-a-backup-file)
+  * [3.18 UC SS\_17: Delete a Backup File](#318-uc-ss_17-delete-a-backup-file)
+  * [3.19 UC SS\_18: Upload a Backup File](#319-uc-ss_18-upload-a-backup-file)
+  * [3.20 UC SS\_19: View the List of Tokens, Keys and Certificates](#320-uc-ss_19-view-the-list-of-tokens-keys-and-certificates)
+  * [3.21 UC SS\_20: View the Details of a Token](#321uc-ss_20-view-the-details-of-a-token)
```
Comment (tmkrepo) [inline doc/UseCases/uc-ss_x-road_use_case_model_for_security_server_management_1.4_Y-883-4.md]: Links broken for 3.21, 3.29, 3.43

File: doc/UseCases/uc-trust_x-road_use_case_model_for_trust_service_management_1.1.1_Y-883-9.md [hunk 5]
```diff
@@ -0,0 +1,1193 @@
...
+25.10.2015   | 0.3     | Renamed *Scope* to *System*. Renamed *User* to *CS administrator*. Use cases updated according to system developments. Some typos and terms fixed. | Riin Saarmäe
+25.10.2015   | 1.0     | Editorial changes made                                         | Riin Saarmäe
+04.11.2015   | 1.1     | UC TRUST\_19 added. Minor corrections made.                    | Riin Saarmäe
+29.08.2017   | 1.2     |  Changed documentation type from docx to md file |   Lasse Matikainen
+
+## Table of Contents
+
+<!-- toc -->
+
+- [1 Introduction](#introduction)
```
Comment (tmkrepo) [inline doc/UseCases/uc-trust_x-road_use_case_model_for_trust_service_management_1.1.1_Y-883-9.md]: None of the table of contents link work

File: doc/UseCases/uc-ss_x-road_use_case_model_for_security_server_management_1.4_Y-883-4.md [hunk 6]
```diff
@@ -0,0 +1,2761 @@
...
+
+2.  System deletes the selected timestamping service from the list of
+    usable timestamping services.
+
+3.  System logs the event “Delete timestamping service” to the audit
+    log.
+
+**Extensions**: -
+
+[]{#__RefHeading__84681_116057917 .anchor}**Related information**:
```
Comment (tmkrepo) [inline doc/UseCases/uc-ss_x-road_use_case_model_for_security_server_management_1.4_Y-883-4.md]: Broken anchor

File: doc/UseCases/uc-gconf_x-road_use_case_model_for_global_configuration_distribution_1.4_Y-883-8.md (hunk 2 above)
Comment (LasseMatikainen) [reply doc/UseCases/uc-gconf_x-road_use_case_model_for_global_configuration_distribution_1.4_Y-883-8.md]: Not sure what you mean here. first extension 3a is missing ul though. if you mean 6a.2 not being 6a.1 it was already that way in the word doc.

File: doc/UseCases/uc-service_x-road_use_case_model_for_service_management_1.6_Y-883-3.md (hunk 3 above)
Comment (tmkrepo) [reply doc/UseCases/uc-service_x-road_use_case_model_for_service_management_1.6_Y-883-3.md]: This is still an issue.

Comment (LasseMatikainen) [reply doc/UseCases/uc-service_x-road_use_case_model_for_service_management_1.6_Y-883-3.md]: https://github.com/vrk-kpa/X-Road/blob/PVAYLADEV-878/doc/UseCases/uc-service_x-road_use_case_model_for_service_management_1.6_Y-883-3.md
It seems to be working

Comment (tmkrepo) [reply doc/UseCases/uc-service_x-road_use_case_model_for_service_management_1.6_Y-883-3.md]: Hmm, somehow saw a version where it didn't, but yes, now it works.

File: doc/UseCases/uc-gconf_x-road_use_case_model_for_global_configuration_distribution_1.4_Y-883-8.md [hunk 7]
```diff
@@ -0,0 +1,1917 @@
...
+    - 4a.1. System displays the error message: “Configuration anchor upload failed: unexpected instance identifier found in anchor”.
+    - 4a.2. System logs the event “Upload configuration anchor failed” to the audit log.
+    - 4a.3. SS administrator selects to reinsert the path to the configuration anchor file. Use case continues from step 3.
+        - 4a.3a. SS administrator selects to terminate the use case.
+
+- 6a. SS administrator cancels the import.
+    - 6a.2. Use case terminates.
+
+- 7a. Downloading of the internal configuration fails.
+    - 7a.1. System displays the error message: “Configuration source cannot b
```
Comment (tmkrepo) [inline doc/UseCases/uc-gconf_x-road_use_case_model_for_global_configuration_distribution_1.4_Y-883-8.md]: I think this should be 2.3.6, now there are two sections numbered 2.3.7.

**Pull request merged**